src/emit.ts

```diff
@@ -11,7 +11,7 @@ const enum DevtoolsHooks {
   COMPONENT_EMIT = 'component:emit'
 }
 
-let events: Events
+let events: Events = {};
```

Comment:
Not directly relevant but as a refactor in the future, I wonder if we should use `WeakMap` instead of a record here :thinking: